### PR TITLE
testutils: remove etcdctl dependency

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: lib/testutils/cleanEtcd.go [hunk 1]
```diff
@@ -15,15 +15,26 @@
...
+}
+
 // CleanEtcd is a utility function to wipe clean "/calico" recursively from etcd.
 func CleanEtcd() {
-	err := exec.Command("etcdctl", "rm", "/calico", "--recursive").Run()
-	if err != nil {
-		log.Println(err)
+	_, err := kapi.Delete(context.Background(), "/calico", &etcdclient.DeleteOptions{Dir: true, Recursive: true})
+	if err != nil && !etcdclient.IsKeyNotFound(err) {
+		panic(err)
```
Comment (gunjan5) [inline lib/testutils/cleanEtcd.go]: can we use `log.Fatalf()` or something similar here instead of panicing?

File: lib/testutils/cleanEtcd.go [hunk 2]
```diff
@@ -15,15 +15,26 @@
...
+	cfg := etcdclient.Config{Endpoints: []string{"http://127.0.0.1:2379"}}
+	c, _ := etcdclient.New(cfg)
+	kapi = etcdclient.NewKeysAPI(c)
+}
+
 // CleanEtcd is a utility function to wipe clean "/calico" recursively from etcd.
 func CleanEtcd() {
-	err := exec.Command("etcdctl", "rm", "/calico", "--recursive").Run()
-	if err != nil {
-		log.Println(err)
```
Comment (gunjan5) [inline lib/testutils/cleanEtcd.go]: I think it's still worth logging even when the error is `KeyNotFound`. WDYT?

**Comment by tomdee** [comment]
OK, updated so behavior is the same as before.

**Comment by gunjan5** [comment]
@tomdee LGTM! squash and merge

**Pull request merged**